.github/workflows/cd.yml

```
@@ -0,0 +1,36 @@
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Kubernetes Kubectl
+      uses: azure/setup-kubectl@v1
+      with:
+        version: 'v1.21.0' # Use the desired version of kubectl
+
+    - name: Set up Kubeconfig
+      env:
+        KUBECONFIG: ${{ secrets.KUBECONFIG }}
+      run: echo "${KUBECONFIG}" > $HOME/.kube/config
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:24: Double quote to prevent globbing and word splitting

(shellcheck)